pygotham/frontend/sponsors.py

```diff
@@ -73,7 +73,7 @@ def edit(pk):
     return render_template('sponsors/edit.html', form=form, sponsor=sponsor)
 
 
-@route(blueprint, '/')
+@route(blueprint, '/', navbar_kwargs={'path': ('Sponsors', 'Sponsors')})
```

Comment:
Ah, yes, if we provide only one item to ``path`` (or just set it to the string), the top tier "Sponsors" item will become the link. Flask-Copilot uses `default_href` as a fallback when a navbar item has no endpoint assigned to it, so all of the top tier links with default to `#` right now.